### PR TITLE
Warning tooltip and color if more than 200 results are found

### PR DESCRIPTION
**Pull request by loicgasser**
https://github.com/geoadmin/mf-geoadmin3/issues/2470

[Test Link](https://mf-geoadmin3.dev.bgdi.ch/gal_2470/?dev3d=true&lang=fr&topic=aviation&bgLayer=ch.swisstopo.pixelkarte-grau&layers=ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bafu.wrz-wildruhezonen_portal,ch.swisstopo.swisstlm3d-wanderwege,ch.bazl.luftfahrthindernis,ch.bafu.schutzgebiete-aulav_jagdbanngebiete&layers_visibility=false,false,false,false,true,true&layers_timestamp=18641231,,,,,&X=163061.70&Y=627775.32&zoom=1&catalogNodes=1379,1381,1514&layers_opacity=1,1,1,1,1,0.75)

warning_tooltip

### CODE REVIEW

**Comment by loicgasser** [comment]
@mariokeusen this is an alternative. I think a popup is too much and that the limitation is per layer not global.

File: src/components/featuretree/FeaturetreeDirective.js [hunk 1]
```diff
@@ -118,11 +118,16 @@ goog.require('ga_map_service');
 
               angular.forEach(tree, function(layerNode, layerBodId) {
                 var oldNode = scope.tree[layerBodId];
+                var className = layerNode.hasMoreResults ? 'warn-color' : '';
+                var label = $sce.trustAsHtml(
+                    gaLayers.getLayer(layerBodId).label +
+                    ' <div class="' + className + '"' +
```
Comment (oterral) [inline src/components/featuretree/FeaturetreeDirective.js]: use a `span` instzead of `div`, the span is displayed `inline` by default

File: src/components/featuretree/partials/featuretree.html [hunk 2]
```diff
@@ -1,8 +1,9 @@
 <ul>
   <li ng-repeat="layer in tree">
-    <label ng-click="layer.open = !layer.open" class="ga-truncate-text">
+    <label ng-click="layer.open = !layer.open" class="ga-truncate-text"
+        title="{{ layer.hasMoreResults ? 'query_alert_more_than_200_results' : '' | translate }}">
       <i ng-class="{'icon-minus': layer.open, 'icon-plus': !layer.open}"></i>
-      {{layer.label}}
+      <div class="layer-name" ng-bind-html="layer.label"></div>
```
Comment (oterral) [inline src/components/featuretree/partials/featuretree.html]: here too a `span`

File: src/components/featuretree/style/featuretree.less [hunk 3]
```diff
@@ -26,6 +26,16 @@
     color: #f00;
   }
 
+  .layer-name {
+    display: inline;
+    * {
+      display: inline;
```
Comment (oterral) [inline src/components/featuretree/style/featuretree.less]: Using a span you don't need these css

Comment (loicgasser) [inline src/components/featuretree/style/featuretree.less]: yup

**Comment by loicgasser** [comment]
updated

File: src/components/featuretree/partials/featuretree.html (hunk 2 above)
Comment (oterral) [inline src/components/featuretree/partials/featuretree.html]: You don't need the `class` attribute

File: src/components/featuretree/FeaturetreeDirective.js [hunk 4]
```diff
@@ -118,11 +118,16 @@ goog.require('ga_map_service');
 
               angular.forEach(tree, function(layerNode, layerBodId) {
                 var oldNode = scope.tree[layerBodId];
+                var className = layerNode.hasMoreResults ? 'warn-color' : '';
```
Comment (oterral) [inline src/components/featuretree/FeaturetreeDirective.js]: can you add the `ga-` prefix please

**Comment by oterral** [comment]
Minor prefix thing otherwise lgtm.

File: src/components/featuretree/FeaturetreeDirective.js (hunk 4 above)
Comment (loicgasser) [inline src/components/featuretree/FeaturetreeDirective.js]: I debated it, why is `.icon-zoom-in` not prefixed?

Comment (oterral) [inline src/components/featuretree/FeaturetreeDirective.js]: It's a font-awesome icon so a fa class css. Will be prefixed in version 4 of fa

Comment (loicgasser) [inline src/components/featuretree/FeaturetreeDirective.js]: ok!

**Comment by gjn** [comment]
I'll squash this and merge.

**Comment by gjn** [comment]
@loicgasser small tip: if you put `fix` in front of the issue in the PR comment, the attributed issue will be auto-closed when PR is merged. https://help.github.com/articles/closing-issues-via-commit-messages/

**Comment by loicgasser** [comment]
thks for the tip

**Pull request merged**